ecs_ec2_user_data.sh.tpl

```diff
@@ -45,3 +45,110 @@ echo "*/5 * * * * root /etc/ecs/cw_ecsmetrics.sh" > /etc/cron.d/ecsmetrics
 } >> /etc/ecs/ecs.config
 start ecs
 echo "Done"
+
+# ASG Auto Assign Elastic IP
+TIMEOUT=20
+PAUSE=5
+
+apt-get update
```

Comment:
As far as I know AWS ECS optimized AMI is is RHEL based, any reason to use ubuntu for that?